pkgs/os-specific/linux/upower/default.nix

```diff
@@ -40,6 +40,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-+C/4dDg6WTLpBgkpNyxjthSdqYdaTLC8vG6jG1LNJ7w=";
   };
 
+  # Remove when this is fixed upstream:
+  # https://gitlab.freedesktop.org/upower/upower/-/issues/214
+  patches = lib.optional (stdenv.hostPlatform.system == "i686-linux")
+    ./i686-test-remove-battery-check.patch;
```

Comment:
FYI: if we apply the patch conditional it will probably be forgotten to be rebased in the future and might not apply eventually.